src/CommitmentsV2/SFA.DAS.CommitmentsV2/Services/CohortDomainService.cs

```diff
@@ -118,6 +118,24 @@ public async Task<Cohort> CreateCohortWithOtherParty(long providerId, long accou
             return accountLegalEntity.CreateCohortWithOtherParty(provider, accountLegalEntity, transferSender, message, userInfo);
         }
 
+        public async Task<Cohort> CreateEmptyCohort(long providerId, long accountId, long accountLegalEntityId, UserInfo userInfo, CancellationToken cancellationToken)
+        {
+            var originatingParty = _authenticationService.GetUserParty();
+
+            if (originatingParty != Party.Provider)
+            {
+                throw new InvalidOperationException($"Only Providers can create empty cohort");
+            }
+
```

Comment:
If we extend the interface, this would get throw by the Employer implementation of ICohortOriginator...